complaint_search/es_builders.py

```diff
@@ -737,6 +737,29 @@ def build(self):
         return aggs
 
 
+class DateRangeBucketsBuilder(BaseBuilder):
+    def build(self):
+        agg = {
+            'dateRangeBuckets': {
+                'aggs': {
+                    'dateRangeBuckets': {
+                        "date_histogram": {
+                            "field": "date_received",
+                            "interval": self.params['trend_interval']
```

Comment:
This line assumes (correctly for now) that trend interval will always be in the params

Defensively, this line could be rewritten as

```python
self.params.get('trend_interval', <some_default_value_here>)
```

Reply:
I would recommend month as it is often our default.